bodhi/tests/server/functional/test_updates.py

```diff
@@ -2520,6 +2520,99 @@ def test_edit_testing_update_reset_karma_with_same_tester(self, publish, *args):
 
     @mock.patch(**mock_valid_requirements)
     @mock.patch('bodhi.server.notifications.publish')
+    def test_has_negative_karma(self, publish, *args):
+        """
+        The test asserts True for has_negative_karma i.e,
+        if update has negative karma
+        """
```

Comment:
The name of the test should reflect the condition it is testing, rather than just the name of the method it is testing. For example, a better name might be ```test_has_negative_karma_with_changed_karma```, since this test is testing a user who gave negative feedback and then changed their mind. Also, the test documentation here is incorrect - this test is asserting False, not true. Make sure that the test documentation covers the condition that is being created below.